src/store/actions.ts

```diff
@@ -76,7 +76,8 @@ export async function getCurrentFirebaseUser(state: State): Promise<State> {
         if (doc.exists) {
             newState.firebaseUser = doc.data();
 
-            newState.firebaseUser.notifications = newState.firebaseUser.notifications.filter(notification => !notification.read);
+            if (newState.firebaseUser.notifications != null)
```

Comment:
Good pickup. I just realised that not all users have the notifications collection. I need to add this to existing accounts. If we can use strict equality for these things (=== instead of == or !=) that would be great.

Reply:
Done!